.travis/script.sh

```diff
@@ -1,40 +1,42 @@
-#!/usr/bin/env sh
-set -v
-
-flake8 --config flake8.cfg || exit 1
-
-result=0
-
+#!/usr/bin/env bash
```

Comment:
sh and bash are quite different. :smile:

Reply:
I personally prefer `sh` since I don't use `bash` but I'm fine with this change.